docs/introduction/integration_test.rst

```diff
@@ -336,6 +336,10 @@ Flow
 6. If the tests fail, you will see it in the the output. For more
    details, you can take a look at the log file (logs to STDOUT as default).
 
+.. note:: User can run several test using several times flag -c. If a folder path is specified,
+a log for each yaml file will be stored. If otherwise a filename is provided, all log information
```

Comment:
I thought it needed several spaces to be considered as a follow-up of the above line. Can you check how the documentation looks like?